components/ui/widgets/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!-- Button styling -->
+    <style name="Mozac.Widgets.NeutralButton" parent="Widget.MaterialComponents.Button.TextButton">
+        <item name="iconTint">@color/button_text_color</item>
+        <item name="iconPadding">8dp</item>
+        <item name="iconGravity">textStart</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:background">@drawable/rounded_button_background</item>
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">48dp</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textAllCaps">false</item>
+        <item name="backgroundTint">@color/grey_button_color</item>
+        <item name="android:textColor">@color/button_text_color</item>
+        <item name="android:letterSpacing">0</item>
+        <item name="fontFamily">@font/metropolis_semibold</item>
+    </style>
+
+    <style name="Mozac.Widgets.DestructiveButton" parent="Mozac.Widgets.NeutralButton">
+        <item name="iconTint">@color/destructive_button_text_color</item>
+        <item name="android:textColor">@color/destructive_button_text_color</item>
+    </style>
+
+    <style name="Mozac.Widgets.PositiveButton" parent="Mozac.Widgets.NeutralButton">
+        <item name="backgroundTint">?accent</item>
```

Comment:
@pocmo how do recommend approaching attributes we have defined in Fenix? Would it make sense to create an `accent` attr here and then we just "override" what that is in Fenix? I don't really want to duplicate a bunch color values here 😕